### ISSUE TITLE
Investigate improvements for Create Community flow

### ISSUE THREAD

**Issue by rbennettcw** [hicommonwealth/commonwealth#4845]
## Acceptance Criteria
- Identify code/UX improvements that can be made for the Create Community flow
- Create a proposal of suggested changes
- Prioritize and create tickets

**Comment by jnaviask** [comment]
Timebox: 3 hours.

**Comment by dillchen** [comment]
@rbennettcw, please see this ticket and include it as part of spike, as it is already completed work. @ianrowan was the implementor) ([link](https://github.com/hicommonwealth/commonwealth/issues/4464)). 

This is from the 2.0 work stream, and a number of UX improvements are implemented.

Please note, within the ticket, that there is a linked PRD ([relinked here](https://www.notion.so/buildcommon/Actually-Good-Community-Creation-20a0b9b071d243ceab7bf47e63ad96be?pvs=4)) that product has _already drafted._

-----
If there are any tickets that include UX improvements, we'll need to get them reviewed by Product before hand.

**Comment by rbennettcw** [comment]
TLDR: There's nothing for me to do here.

- There are already requirements defined for improving the community creation flow (ticket linked by Dillon)
  - Ian has a WIP PR based on those requirements
- From a code standpoint, I don't see any major improvements to make to the current code on the master branch
- In terms of UX, it's best to let Product create the tickets in the first place

**Comment by rbennettcw** [comment]
@jnaviask I left the comment above to clarify why I closed the ticket.

If the ticket should be reopened, please clarify what the acceptance criteria of the ticket should be.

**Comment by jnaviask** [comment]
Sounds good. Thank you @rbennettcw.
